.github/workflows/AMAZON2.tfvars

```diff
@@ -0,0 +1,9 @@
+# Amazon Linux 2
```

Comment:
This will be moved to that workflow and a new config we have, due to the way we have restricted direct changes to the workflow files we have to go through a few PRs ensuring pipelines pass before files are commited. Its on its way. :)